ingestion/src/metadata/ingestion/source/database/mssql/__init__.py

```diff
@@ -0,0 +1,19 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+Module constants
+"""
+
+
+MSSQL_GET_DATABASE = """
```

Comment:
let's not add the constants here. For other sources we are using a `queries` module to add this info. IMO it helps point to what we need to know

Reply:
Sure will add this to the `queries` File.